lib/x11utils.pm

```diff
@@ -315,10 +315,11 @@ sub select_user_gnome {
 
  turn_off_kde_screensaver();
 
-Turns off the screensaver on KDE
+Turns off the screenlocker and energy saving in the Plasma desktop.
 
 =cut
 sub turn_off_kde_screensaver {
+    # Disable screenlocker
```

Comment:
I really like this comment. Way to go Anton, this makes it clear what happens! :+1:

Reply:
As an alternative better extract the according code into a function

Reply:
Is that what you had in mind? TBH I do not like it this way but only aesthetically :).